src/chronometer-centiseconds.js

```diff
@@ -1,37 +1,49 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId = setInterval (() => {
+      this.currentTime++; 
+  
+      if(printTimeCallback) {
+    printTimeCallback()
+      }
+     }, 10);
   }
 
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 6000);
   }
 
   getSeconds() {
-    // ... your code goes here
+    return Math.floor(this.currentTime * 0.01) % 60; 
   }
 
   getCentiseconds() {
-    // ... your code goes here
+    return Math.floor(this.currentTime % 100);
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    if (value >= 10) {
+      return `${value}`
+    }
+    else {
+      return `0${value}`
+    }
   }
 
   stop() {
-    // ... your code goes here
+    clearInterval(this.intervalId); 
   }
 
   reset() {
-    // ... your code goes here
+    return this.currentTime = 0;
   }
 
   split() {
-    // ... your code goes here
+    return `${this.computeTwoDigitNumber(this.getMinutes())}:${this.computeTwoDigitNumber(this.getSeconds())}.${this.computeTwoDigitNumber(this.getCentiseconds())}`
```

Comment:
Everything is correct. Let me just show you another way to do it. Here, you could've created variables to store this and returned the variables, like:

```
let minutes = this.computeTwoDigitNumber(this.getMinutes());
let seconds = this.computeTwoDigitNumber(this.getSeconds());
let centiseconds = this.computeTwoDigitNumber(this.getCentiseconds());
return `${minutes}:${seconds}.${centiseconds}`;
```